### ISSUE TITLE
New website is not rendered correctly in mobile view

### ISSUE THREAD

**Issue by alfonsogarciacaro** [fable-compiler/fable-compiler.github.io#101]
I believe this is because we're missing the responsive tag:

```html
<meta name="viewport" content="width=device-width, initial-scale=1">
``` 

I cannot find an HTML template in the repo so I guess it's in Nacara? Can we add the responsive tag to the default layout there? @MangelMaxime

**Comment by MangelMaxime** [comment]
Yes Nacara generate everything even the html element. It is possible that I forgot to add it and when using browser resize to emulate mobile I didn't saw this problem coming.

**Comment by MangelMaxime** [comment]
I just released version 1.0.0-beta-005 of nacara-layout-standard so upgrading this dependency should fix the problem.

Sorry, I am on vacation with limited internet access.

**Comment by MangelMaxime** [comment]
It seems fixed for me on my phone.

Please feel free to re-open if needed
